build/ci/set-up-ops-manager.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+# Copyright 2021 MongoDB Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+while getopts 'h:' opt; do
+  case ${opt} in
+  h) hostsFile="${OPTARG}" ;; # Output of Evergreen host.list
+  *) exit 1 ;;
+  esac
+done
+
+host=$(
+  cat <<EOF | python - "${hostsFile}"
+import sys
+import json
+with open(sys.argv[1]) as hostsfile:
+    hosts = json.load(hostsfile)
+    print(hosts[0]["dns_name"])
+EOF
+
+)
+
+cd ../..
+
+MCLI_OPS_MANAGER_URL="http://${host}:9080/"
+
+echo "set service"
+  ./bin/mongocli config set service "${ops_manager_service:?}"
+
+echo "set ops_manager_url"
+  ./bin/mongocli config set ops_manager_url "${MCLI_OPS_MANAGER_URL}"
+
+echo "generate password for owner"
+password=$(date +%s | sha256sum | base64 | head -c 8)0_
+
+echo "generate email for owner"
+email=$(date +%s | sha256sum | base64 | head -c 8)@ops-manager-team.com
+
+echo "create first user"
+MCLI_PRIVATE_API_KEY=$(./bin/mongocli om owner create --firstName evergreen --lastName evergreen --email "${email}" --password "${password}" -o="go-template={{.APIKey}}")
+
+echo "set public_api_key"
+./bin/mongocli config set public_api_key "${email}"
+
+echo "set private_api_key"
+./bin/mongocli config set private_api_key "${MCLI_PRIVATE_API_KEY}"
+
+echo "create organization"
+MCLI_ORG_ID=$(./bin/mongocli iam organizations create myOrg -o="go-template={{.ID}}")
+
+echo "create project"
+MCLI_PROJECT_ID=$(./bin/mongocli iam projects create myProj --orgId "${MCLI_ORG_ID}" -o="go-template={{.ID}}")
+
+
+cat <<EOF > "${XDG_CONFIG_HOME}/mongocli.toml"
```

Comment:
I am not following. Could you give me more information on this, please (maybe with an example)? I am setting all these variables (`MCLI_ORG_ID`, `MCLI_PROJECT_ID`, `MCLI_PRIVATE_API_KEY` etc) so that I can create the toml.

Reply:
you do a lot of `/bin/mongocli config set <something>` those are creating a new `mongocli.toml` that at the end you're not using as you're generating a new file here.
If the reason you are doing `config set` is to have settings for `./bin/mongocli om owner create` and the rest, you don't have to

you can do 
```bash
export MCLI_OPS_MANAGER_BASE_URL="${MCLI_OPS_MANAGER_URL}"
export MCLI_SERVICE="ops-manager"
./bin/mongocli om owner create
```
Or even better now that I write this, rename `MCLI_OPS_MANAGER_URL` to the correct name and you don't even need to export it here 
Also this is valid tho not better in case of secrets
```bash
MCLI_OPS_MANAGER_BASE_URL="value" MCLI_SERVICE="value" mongocli [command]
```
And no need to export, again exporting is fine just shoing you examples that you don't need to call `config set`

You can keep exporting as needed instead of `config set`